library/src/main/kotlin/com/liferay/vulcan/consumer/extensions/Date+.kt

```diff
@@ -0,0 +1,14 @@
+package com.liferay.vulcan.consumer.extensions
+
+import java.text.DateFormat
+import java.util.*
+
+fun Date.shortFormat() = format(DateFormat.getDateInstance(DateFormat.SHORT, Locale.US))
```

Comment:
I could add the Locale as a parameter with default option `locale: Locale = Locale.US`

Will that be ok?